app/src/main/kotlin/com/wire/android/ui/userprofile/other/OtherUserProfileScreen.kt

```diff
@@ -325,7 +355,7 @@ fun OtherProfileScreenContent(
                 onOpenConversation
             )
         },
-        isSwipeable = state.connectionState == ConnectionState.ACCEPTED
+        isSwipeable = state.connectionState != ConnectionState.BLOCKED
```

Comment:
This is another issue discovered when the content is not swipeable, we should disable that only for blocked state